packages/react-native-bridge/android/publish-aztec-and-bridge.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# 1. Verify the version argument is passed. We use the same version for react-native-aztec and react-native-bridge libraries.
+VERSION=$1
+if [[ -z $VERSION ]]; then
+    echo "This script requires the Bintray version to be passed as an argument."
+    echo "Example usage: './publish-aztec-and-bridge.sh \$VERSION'"
+    exit 1
+fi
+
+# 2. Check if we can call gradlew successfully
+./gradlew --help > /dev/null 2>&1
+gradlew_exists=$?
+if [ $gradlew_exists -ne 0 ]; then
+    echo "Gradle wrapper is not available. Make sure to run this script from '/packages/react-native-bridge/android'"
```

Comment:
Curious why we're just checking whether gradlew can be invoked instead of checking that we're in `packages/react-native-bridge/android`. I don't think this is a very big deal, I'm more just curious if perhaps there are some considerations I'm overlooking.

Reply:
That's a good question. Let me see if I can do that instead, particularly, let me see if I can also verify whether we are in a submodule `gutenberg-mobile` per [this comment](https://github.com/WordPress/gutenberg/pull/29488#discussion_r589056149).